ledger/sandbox-classic/src/main/scala/platform/sandbox/package.scala

```diff
@@ -11,7 +12,7 @@ package object sandbox {
   private[sandbox] val Name = LedgerName("Sandbox")
 
   val DefaultConfig: SandboxConfig = SandboxConfig.defaultConfig.copy(
-    seeding = None,
+    seeding = Some(Seeding.Strong),
```

Comment:
:clap: 

This is the default though, so you can just delete the line.

Reply:
I reverted back to None as it is the default Bernhard want.